src/test/resources/org/jenkinsci/plugins/scriptsecurity/sandbox/groovy/SandboxInterceptorTest/all.groovy

```diff
@@ -26,6 +26,13 @@ assert arr == [0];
 assert true in [true, false];
 (1..3).each {};
 [1..3]*.toString();
+
+// convert env variable to docker env parameters
+assert "test=test -e test2=test2" == ["test=test","test2=test2"].inject([]) { result, entry ->
```

Comment:
OK, I thought that the added elements should be tested in this file. I revert it.